Marlin/Marlin_main.cpp

```diff
@@ -457,7 +457,7 @@ static int serial_count = 0;
 // Inactivity shutdown
 millis_t previous_cmd_ms = 0;
 static millis_t max_inactive_time = 0;
-static millis_t stepper_inactive_time = (DEFAULT_STEPPER_DEACTIVE_TIME) * 1000UL;
+millis_t stepper_inactive_time = (DEFAULT_STEPPER_DEACTIVE_TIME) * 1000UL;
 
```

Comment:
> IN SCOPE of the set static variable and therefore cannot be changed

Static variables can be changed. Perhaps you're thinking of `const`?? The only thing `static` does is make the compiler bark if you try to use `extern millis_t stepper_inactive_time` in another compilation unit. It tells the compiler that this variable is only visible inside `Marlin_main.cpp`.